### ISSUE TITLE
change in si.add API or config definition?

### ISSUE THREAD

**Issue by eklem** [fergiemcdowall/search-index#265]
From v0.7.6, what's the change that makes `search-index` spit out this message? Comes after the whole batch is sent to indexing.

```
{"name":"search-index","hostname":"Oles-MacBook-Pro.local","pid":50297,"level":30,"err":{"message":"Key not found in database [RI￮*￮st\\nodemodules\\hawk\\nodemodules\\cryptiles\\makefile￮￮]","name":"NotFoundError","stack":"NotFoundError: Key not found in database [RI￮*￮st\\nodemodules\\hawk\\nodemodules\\cryptiles\\makefile￮￮]\n    at /Users/eklem/node_modules/search-index-indexer/node_modules/levelup/lib/levelup.js:217:15"},"msg":"Key not found in database [RI￮*￮st\\nodemodules\\hawk\\nodemodules\\cryptiles\\makefile￮￮]","time":"2016-03-30T09:48:46.894Z","v":0}
```

It was fine again after I changed the dependency in package.json from:
```
    "dependencies": {
        "search-index": "~0.7.6"
    }
```
to:
```
    "dependencies": {
        "search-index": "0.7.6"
    }
```
That's why I'm guessing a change in some API.

**Comment by eklem** [comment]
This is [the actual add-stuff I do](https://github.com/eklem/search-index-indexer/blob/master/index.js#L57-L69), and these are the [config](https://github.com/eklem/search-index-indexer/blob/master/config.json) and [data](https://github.com/eklem/search-index-indexer/blob/master/data.json).

**Comment by fergiemcdowall** [comment]
Ahh- I think there has been no change to the functionality- some of the log levels have been jumbled up. The message you got should only be spit out if you are running your logs in debug mode

**Comment by eklem** [comment]
Ok, this is the info-level. Looks the same as the debug-level.

**Comment by fergiemcdowall** [comment]
@eklem did you get this working in the end? Does it need to be fixed?

**Comment by eklem** [comment]
[I did a workaround](https://github.com/eklem/search-index-indexer/commit/339b5a105acaf7a8a2601185e56a06af0ea55ef9) changing log level, but the bug is still there, as far as I know.

**Comment by eklem** [comment]
Still need to be fixed, checked today.

**Comment by eklem** [comment]
Fixed as of version 0.8.10
